src/interfaces/HposInterface.js

```diff
@@ -246,35 +231,48 @@ const HposInterface = {
   },
 
   getHoloFuelProfile: async () => {
-    const {
-      agent_address: agentAddress,
-      nickname,
-      avatar_url: avatarUrl
-    } = await hposHolochainCall({
-      method: 'post',
-      path: '/zome_call'
-    })({
-      appId: CORE_APP_ID,
-      roleId: 'holofuel',
-      zomeName: 'profile',
-      fnName: 'get_my_profile',
-      payload: null
-    })
+    try {
+      const params = {
+        appId: CORE_APP_ID,
+        roleId: 'holofuel',
+        zomeName: 'profile',
+        fnName: 'get_my_profile',
+        payload: null
+      }
+
+      const {
+        agent_address: agentAddress,
+        nickname,
+        avatar_url: avatarUrl
+      } = await hposHolochainCall({
+        method: 'post',
+        path: '/zome_call',
+        params
+      })
 
-    return { agentAddress: Uint8Array.from(agentAddress.data), nickname, avatarUrl }
+      return { agentAddress: Uint8Array.from(agentAddress.data), nickname, avatarUrl }
+    } catch (error) {
+      return {
+        nickname: null,
+        avatarUrl: null
+      }
```

Comment:
```suggestion
      return {}
```
Forgot to reset this to just an empty obj.

@mateuszRybczonek - I know you're already testing this / working on this branch, so I didn' want to push up a change. Feel free to accept this should you like / when you're ready.